try:
    print(num)
except NameError:
    print("Name Error exception is thrown, num is not defined")